manage.py

```diff
@@ -63,6 +63,28 @@ def runserver(info, host, port, debugger):
                extra_files=reload_on_files)
 
 
+@cli.command()
+def check_highlevel():
```

Comment:
This is a good start, a few things here:
 * We explicitly add a `name` parameter to `cli.command` in this file. We should do the same here.
 * the name of this method should more accurately indicate what we are checking. It should include a docstring so that the automatically generated help for manage.py shows what it does.
 * I think it would be a good idea to put these commands behind a subcommand, like we do with `dump`, later in this file. This means that we can have two commands: `manage.py highlevel show_failed_rows` and `manage.py highlevel remove_failed_rows`

Reply:
For this, i removed the commands check_highlevel and clean_highlevel from manage.py and added them as subcommands in manage_highlevel.py file